apps/docs/content/components/CTABanner/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: CTA banner
+description: Use CTA banner to draw attention to or create urgency around a user action.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing the CTA banner anatomy with a heading, a description and a button group.](https://github.com/primer/brand/assets/912236/779c909d-21eb-42b8-b3dd-963e0113102c)
+
+- Heading: The main title of the Pillar. It should be short and concise.
+- Description: Short text that extends the information provided by the heading.
+- Button group: The main and secondary actions the user should take on the page
+
+## Usage
+
+Use the CTA banner to draw attention around the main user actions. The CTA banner should be used sparingly and is typically placed at the bottom of the page to close the user journey,
+make a short summary of the page, and to persist the main actions. The CTA banner can also be used between other sections on lengthy pages.
+
+The CTA banner can include primary and secondary actions. These actions are rendered as a [button group](/components/ButtonGroup) and follow the same guidelines as the button group component.
+
+The actions used in a CTA banner are normally the same as the ones used on the [hero](/components/Hero) component on the top of the page. This repetition provides consistency and helps the user to remember what the main actions are.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/952c6855-139b-42c1-a5df-ae3574c02f3c" />
+    <Caption>
+      Use the CTA banner on the bottom of the page to close the user journey and
+      reinforce the main actions.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/389a4458-a558-4f8f-9656-d1aa51c82b3e" />
+    <Caption>
+      Don't use the CTA early on in the user journey, use a hero or a section intro
+      component instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Align
+
+The CTA banner elements can be aligned to the start or center of the page. By default, the CTA banner is aligned to the start of the page. Use the center alignment if context calls for it.
```

Comment:
> Use the center alignment if context calls for it.

Can we provide examples of this context? As a reader, this feels a little unclear, but might just be me. I think a do/dont example could be helpful.